docker-compose.yml

```diff
@@ -168,13 +198,13 @@ services:
       - /dev/shm:/dev/shm
     networks:
       - testing
-    ports:
-      - 5900:5900
```

Comment:
This is probably so you can start multiple instances right? Being able to vnc into the container is really nice though. Could we maybe automatically let docker just set this to `5900 + $whateverNextPortIsFree`? Or alternatively, can grid start multiple browsers in one container, and then we only have vnc there?

But I'm okay sacrificing vnc for speed, so if it's too hard just leave it out.

Reply:
Yes the grid supports starting multiple sessions within the same container, although not recommended. I was indeed going for the `docker-compose scale chrome=XX`, but if we find that the multiple-session within one container works for us, even better.

Reply:
I've never tried it and I'm not sure what exactly it is  used for, but here it shows you can set a range for the host ports (see the last example int he short syntax list)

https://docs.docker.com/compose/compose-file/#ports

Perhaps that would enable port-autoselecting behaviour